<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="register_label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fname_signup">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="phone_signup">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>152</y>
      <width>371</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Phone Number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lname_signup">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>181</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Password_signup">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>271</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="username">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="repeatpasswprd_signup">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Repeat Passwprd</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="city_signup">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>384</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>City</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="email_signup">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>442</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>37</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="midLineWidth">
     <number>4</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>307</x>
      <y>34</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="midLineWidth">
     <number>4</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="date_signup">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>496</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Date ( yyyy/mm/dd )</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Submit_signup">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>552</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>479</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
